### PR TITLE
CLI acceptance tests for parallel executions

### PR DESCRIPTION
**Pull request by jframos**
#### Reviewers

@flopezag @chemaper 
#### Description

This PR solves _CLAUDIA-4545/CLAUDIA-4549_: Acceptance tests for GlanceSync Parallel Execution:
-> Added new test feature for CLI parallel option with some scenarios already defined but executed in a parallel way.
-> Added new operations in Remote GlanceSync CMD Client to manage output log files and its content.
-> Refactor: Common assertions used by different methods.
#### Testing

Locally.

### CODE REVIEW

File: tests/acceptance/features/glancesync/steps/parallel.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+# Get logger for Behave steps
+__logger__ = logging.getLogger("parallel_steps")
+
+__dataset_utils__ = DatasetUtils()
+
+
+@step(u'files are created with output logs')
+def file_created_output_logger(context):
+    result = context.glancesync_client.get_output_log_list()
```
Comment (flopezag) [inline tests/acceptance/features/glancesync/steps/parallel.py:48]: Provide a description of the tests

File: tests/acceptance/features/glancesync/steps/synchronization.py [hunk 2]
```diff
@@ -22,19 +22,15 @@
 # contact with opensource@tid.es
```
Comment (flopezag) [inline tests/acceptance/features/glancesync/steps/synchronization.py:22]: Add description to the test methods.

**Comment by jframos** [comment]
Description added to the new steps file: `parallel.py`

**Comment by flopezag** [comment]
Pending description in synchronization.py?

**Comment by jframos** [comment]
in `synchronization.py` and almost all step implementations we have... Should we fix them in this PR?

**Comment by flopezag** [comment]
For obvious reason the tests/acceptance/features/glancesync/steps/synchronization.py have to be cleared described in this PR, sorry.

**Comment by flopezag** [comment]
LGTM

**Pull request merged**